### DISCUSSION TITLE
Disable showing completions in minibuffer

### DISCUSSION THREAD

**Discussion by oscu0** [company-mode/company-mode#1192] [Q&A] [answered]
Is it possible to disable displaying completions in the mini buffer, i.e. only use the pop-up? I think it's the front-end setting, but mine only contains tooltip and preview.

**Comment by dgutov** [comment, marked as answer]
You're probably referring to `company-echo-metadata-frontend`. It's in `company-frontends`, try removing it.

**Comment by philippludwig** [comment]
Seems like `company-echo-metadata-frontend` does not exist anymore. But a good workaround seems to be `(setq company-echo-delay 10000)`.

**Reply by dgutov** [reply]
Hey, `company-echo-metadata-frontend` does exist (how do you figure it doesn't?), but your snippet should indeed delay the echo by a lot.

**Reply by philippludwig** [reply]
Because my Emacs says so: `Debugger entered--Lisp error: (void-variable company-echo-metadata-frontend)`

**Reply by dgutov** [reply]
It's not a variable but a function. Emacs Lisp is Lisp-2 so those have separate namespaces.

You can check using `C-h f` (`M-x describe-function`).
